includes/classes/ia.core.users.php

```diff
@@ -1059,4 +1065,16 @@ public function getFavorites($ids)
 
         return $this->coreSearch($where, 0, 50, null)[1];
     }
+
+    private function addIpAddressMember()
+    {
+        $data = [
+            'member_name' => $_SESSION['user']['username'],
+            'ip_address' => $_SERVER['REMOTE_ADDR'],
+            'user_agent' => $_SERVER['HTTP_USER_AGENT'],
+            'entry_date' => date( 'Y-m-d h:i:s', time())
```

Comment:
Not sure, but looks like we have  iaDb method for this. I might be wrong.  But I'm sure we have constants for this format. Check iaDb